orderbook/openapi.yml

```diff
@@ -629,6 +654,33 @@ components:
           description: "Hash of the corresponding settlement transaction containing the trade (if available)."
           $ref: "#/components/schemas/TransactionHash"
           nullable: true
+      required:
+        - blockNumber
+        - logIndex
+        - orderUid
+        - owner
+        - signed
```

Comment:
naming nit: is "signed" the correct status - would maybe active/valid/tradable be better? I feel like once the order is used it's still signed.

Reply:
I used "signed" since this is the name of the variable in the SC event. I agree it's not good naming for the API itself, but after implementing [Nick's comment](https://github.com/gnosis/gp-v2-services/pull/996#discussion_r693706919) I think the remaining uses of `signed` would be appropriate.